test/vm.install

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh -eux
+
+TEST_SOURCE="$1"
+
+# Grow root partition to make room for images. This only works on Fedora right now.
+echo ", +" | sfdisk -N 2 -f /dev/vda
+partprobe
+pvresize /dev/vda2
+lvresize fedora/root -l+100%FREE -r
+
+tmpdir=$(mktemp -d $PWD/srpm-build.XXXXXX)
+tar -xaf $TEST_SOURCE -O $(basename --suffix .tar.gz $TEST_SOURCE)/lorax.spec > $tmpdir/lorax.spec
+rpmbuild -bs \
```

Comment:
FTR, the *source* rpm can easily be built in the container, outside of the VM, like we do in starter-kit and friends. This works as well, but the .srpm is an useful developer artifact to have outside, for copr, mock builds, inspection, etc.